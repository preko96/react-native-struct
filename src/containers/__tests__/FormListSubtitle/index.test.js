import React from 'react'
import FormsListSubtitle from '../../FormsListSubtitle/index'

// import configureMockStore from 'redux-mock-store'
// import thunkMiddleware from 'redux-thunk'
// export const mockStore = configureMockStore([thunkMiddleware])

it('renders without crashing', () => {
	jest.mock('react-native', () => require('react-native-mock-render/mock'), {virtual: true})
	const item = {
		completedForms: 0,
		totalForms: 1
	}
	const wrapper = mount(<FormsListSubtitle item={item} />)
	wrapper.mount()
	
	expect(wrapper).toMatchSnapshot()
})