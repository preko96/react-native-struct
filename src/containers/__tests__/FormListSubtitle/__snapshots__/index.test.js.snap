// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<FormsListSubtitle
  item={
    Object {
      "completedForms": 0,
      "totalForms": 1,
    }
  }
>
  <Styled(ViewNB)
    style={
      Object {
        "marginLeft": 10,
        "marginTop": 5,
      }
    }
  >
    <ViewNB
      style={
        Array [
          Object {},
          Object {
            "marginLeft": 10,
            "marginTop": 5,
          },
        ]
      }
    >
      <Component
        style={
          Array [
            Object {},
            Object {
              "marginLeft": 10,
              "marginTop": 5,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {},
              Object {
                "marginLeft": 10,
                "marginTop": 5,
              },
            ]
          }
        >
          <Styled(ViewNB)
            style={
              Object {
                "backgroundColor": "red",
                "borderRadius": 13,
                "height": 13,
                "width": 13,
              }
            }
          >
            <ViewNB
              style={
                Array [
                  Object {},
                  Object {
                    "backgroundColor": "red",
                    "borderRadius": 13,
                    "height": 13,
                    "width": 13,
                  },
                ]
              }
            >
              <Component
                style={
                  Array [
                    Object {},
                    Object {
                      "backgroundColor": "red",
                      "borderRadius": 13,
                      "height": 13,
                      "width": 13,
                    },
                  ]
                }
              >
                <View
                  style={
                    Array [
                      Object {},
                      Object {
                        "backgroundColor": "red",
                        "borderRadius": 13,
                        "height": 13,
                        "width": 13,
                      },
                    ]
                  }
                />
              </Component>
            </ViewNB>
          </Styled(ViewNB)>
          <Styled(ViewNB)
            style={
              Object {
                "alignSelf": "flex-end",
                "left": 20,
                "position": "absolute",
                "top": -1,
              }
            }
          >
            <ViewNB
              style={
                Array [
                  Object {},
                  Object {
                    "alignSelf": "flex-end",
                    "left": 20,
                    "position": "absolute",
                    "top": -1,
                  },
                ]
              }
            >
              <Component
                style={
                  Array [
                    Object {},
                    Object {
                      "alignSelf": "flex-end",
                      "left": 20,
                      "position": "absolute",
                      "top": -1,
                    },
                  ]
                }
              >
                <View
                  style={
                    Array [
                      Object {},
                      Object {
                        "alignSelf": "flex-end",
                        "left": 20,
                        "position": "absolute",
                        "top": -1,
                      },
                    ]
                  }
                >
                  <Styled(Text)
                    style={
                      Object {
                        "color": "#999",
                        "fontSize": 13,
                      }
                    }
                  >
                    <Text
                      style={
                        Array [
                          Object {
                            "color": "#000",
                            "fontFamily": "System",
                            "fontSize": 16,
                          },
                          Object {
                            "color": "#999",
                            "fontSize": 13,
                          },
                        ]
                      }
                      uppercase={false}
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            },
                            Object {
                              "color": "#999",
                              "fontSize": 13,
                            },
                          ]
                        }
                        uppercase={false}
                      >
                        <Text
                          accessible={true}
                          allowFontScaling={true}
                          ellipsizeMode="tail"
                          style={
                            Array [
                              Object {
                                "color": "#000",
                                "fontFamily": "System",
                                "fontSize": 16,
                              },
                              Object {
                                "color": "#999",
                                "fontSize": 13,
                              },
                            ]
                          }
                          uppercase={false}
                        >
                          0
                          /
                          1
                           Forms Completed
                        </Text>
                      </Text>
                    </Text>
                  </Styled(Text)>
                </View>
              </Component>
            </ViewNB>
          </Styled(ViewNB)>
        </View>
      </Component>
    </ViewNB>
  </Styled(ViewNB)>
</FormsListSubtitle>
`;
